user-service/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# https://github.com/awslabs/aws-lambda-web-adapter/tree/main/examples/expressjs
+FROM public.ecr.aws/docker/library/node:20
+
+# expose 8001 port for access to the container
+ENV PORT=8001
+EXPOSE 8001
+
+# copy all files into the Docker container
+COPY . .
+
+# install node libraries
+RUN npm install
+
+# execute the server
+CMD ["node", "server.js"]
```

Comment:
Take note that the usual `npm run dev` command is not used here as no `.env` file is created here. Environment variables are defined in the AWS ECS Task Definition through the online console. Since files in the Docker container will not change, I have opted to use the `node` command rather than the `nodemon` command as well.